modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/util/DDMFormValuesToFieldsConverterTest.java

```diff
@@ -544,6 +577,33 @@ protected void testField(
 			expectedPtValues, field.getValues(LocaleUtil.BRAZIL));
 	}
 
+	private void _addDDMFormFieldValue(
+		DDMFormValues ddmFormValues, String value) {
```

Comment:
I think this method is too specific to have a generic name or parameters.

I see two possible renames:

1. Method name > _addFieldsetDDMFormFieldValue()
2. Parameter name > String nestedValue (?)

I recommend renaming only the method name because we know that the fieldset itself does not contain a value, so the second parameter is the value for the nested field.

Reply:
I changed the method name